### PR TITLE
Feature skeleton for profile2

### PR DESCRIPTION
**Pull request by hiroyalty**
The Front end skeleton for editing User profile, the form use hooks to maintain state and you can console.log the submitted user details.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -11,12 +13,14 @@ import "./App.css";
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
+      <MuiPickersUtilsProvider utils={DateFnsUtils}>
       <BrowserRouter>
         <Switch>
           <Route path="/signup" component={Register} />
           <Route path="/" component={LandingPage} />
```
Comment (langdin) [inline client/src/App.js:20]: I think you need to use -> exact path="/" <-

File: client/src/pages/UserProfile.js [hunk 2]
```diff
@@ -1,53 +1,278 @@
...
+  const [emailAddress, setEmailAddress] = useState('')
+  const [phoneNumber, setPhoneNumber] = useState('')
+  const [addressOne, setAddressOne] = useState('')
+  const [addressTwo, setAddressTwo] = useState('')
+  const [city, setCity] = useState('')
+  const [province, setProvince] = useState('')
+  const [zipCode, setZipCode] = useState('')
+  const [country, setCountry] = useState('')
+  const [aboutme, setAboutMe] = useState('Tell Us about yourself')
+
```
Comment (langdin) [inline client/src/pages/UserProfile.js:47]: maybe use one useState for profile object with all fields together?

File: client/src/pages/UserProfile.js [hunk 3]
```diff
@@ -1,53 +1,278 @@
...
+    setGender(event.target.value);
+  };
+
+  const handleSubmitForm = (event) => {
+    event.preventDefault()
+    const userProfile = {
+      firstName,
+      lastName,
+      gender,
+      DateOfBirth: selectedDate,
```
Comment (bonnieli) [inline client/src/pages/UserProfile.js:63]: `dateOfBirth`

Comment (bonnieli) [reply client/src/pages/UserProfile.js:63]: follow camelCase

File: client/src/pages/UserProfile.js [hunk 4]
```diff
@@ -1,53 +1,278 @@
...
+          fullWidth
+          autoComplete="billing address-line1"
+          value={addressOne}
+          onChange={event => {
+            setAddressOne(event.target.value)
+          }}
+        />
+      </Grid>
+      <Grid item xs={12}>
+        <TextField
```
Comment (bonnieli) [inline client/src/pages/UserProfile.js:188]: you can consider making these TextFields into its own component, taking in their respected props

File: client/src/pages/UserProfile.js (hunk 2 above)
Comment (bonnieli) [reply client/src/pages/UserProfile.js:47]: great point - i'm personally a fan of this

**Pull request merged**